eng/tools/typespec-validation/src/rules/sdk-tspconfig-validation.ts

```diff
@@ -252,6 +264,29 @@ export class TspConfigTsMgmtModularPackageNameMatchPatternSubRule extends TspCon
   }
 }
 
+// ----- TS data plane sub rules -----
+export class TspConfigTsDpPackageDirectorySubRule extends TspconfigEmitterOptionsSubRuleBase {
+  constructor() {
+    super("@azure-tools/typespec-ts", "package-dir", new RegExp(/^(?:[a-z]+-)*rest$/));
+  }
+  protected skip(config: any,  folder: string) {
+    return skipForNonModularOrManagementPlaneInTsEmitter(config,folder);
```

Comment:
```suggestion
    return skipForManagementPlane(folder);
```
this is only used for dataplane
management plane has its own rule https://github.com/Azure/azure-rest-api-specs/blob/main/eng/tools/typespec-validation/src/rules/sdk-tspconfig-validation.ts#L237